core/src/main/java/com/gentics/mesh/core/endpoint/RolePermissionHandlingEndpoint.java

```diff
@@ -77,5 +78,21 @@ protected void addRolePermissionHandler(String uuidParameterName, String uuidPar
 			String uuid = rc.request().getParam(uuidParameterName);
 			crudHandler.handleRevokePermissions(ac, uuid);
 		});
+
```

Comment:
HTTP standard has no directions over DELETE methods with body, so the client generator fails on the method above, as not following the standard.